content/events/cloud-engineering-summit/index.md

```diff
@@ -0,0 +1,27 @@
+---
+# Name of the event.
+title: "Cloud Engineering Summit"
+
+# Events with unlisted as true will not be shown on the event list
+unlisted: false
+
+# Events with external registrations should not be indexed
+# and have redirect to the external registration page.
+block_external_search_index: true
+redirect_to: "https://cloudengineering.heysummit.com/"
```

Comment:
Not part of this PR, but as a general rule, we should probably tack on a `rel="noopener noreferrer"` to any links that open new tabs and aren't on domains we ultimately manage ourselves. (Looks like the webinar page covers this for the text link; the image looks like it doesn't target a new window.) It's worth taking a quick pass to make sure these layouts handle this properly (as admittedly, this slipped under my radar as well).